addons/medical_engine/functions/fnc_conditionIsConsideredPlayer.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: RusComBear
+ * Condition for unit to be considered player for medical system
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_medical_statemachine_fnc_conditionIsConsideredPlayer
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+if (isPlayer _unit) then {
+    true
+} else {
+    switch (EGVAR(medical,considerSelectAIPlayersForMedical)) do {
+	    case 0: {false};
+	    case 1: { 
+            if (isMultiplayer) then {
+                [false, true] select (_unit in playableUnits);
```

Comment:
```suggestion
                _unit in playableUnits // return
```

Reply:
These are even better. Too obvious for me to realize.